webapp/channels/src/components/advanced_create_comment/advanced_create_comment.test.tsx

```diff
@@ -1074,57 +1110,57 @@ describe('components/AdvancedCreateComment', () => {
                 onSubmit,
             };
 
-            const wrapper = shallow(
+            const wrapper = shallow<AdvancedCreateComment>(
                 <AdvancedCreateComment {...props}/>,
             );
 
-            wrapper.instance().handleSubmit({preventDefault});
+            wrapper.instance().handleSubmit(preventDefaultObject);
             expect(onSubmit).toHaveBeenCalled();
-            expect(wrapper.state('draft').props).toBe(undefined);
+            expect(wrapper.state('draft')!.props).toBe(undefined);
         });
     });
 
     test('removePreview should remove file info and upload in progress with corresponding id', () => {
         const onUpdateCommentDraft = jest.fn();
-        const draft = {
+        const draft: any = {
             message: 'Test message',
             uploadsInProgress: [4, 5, 6],
             fileInfos: [{id: 1}, {id: 2}, {id: 3}],
         };
-        const props = {...baseProps, draft, onUpdateCommentDraft};
+        const props: any = {...baseProps, draft, onUpdateCommentDraft};
 
-        const wrapper = shallow(
+        const wrapper = shallow<AdvancedCreateComment>(
             <AdvancedCreateComment {...props}/>,
         );
 
         wrapper.setState({draft});
 
-        wrapper.instance().removePreview(3);
+        wrapper.instance().removePreview(3 as any);
```

Comment:
I don't know why the file IDs and the IDs passed in as `uploadsInProgress` are numbers in so many of these tests since they should've always been strings. I see you changed a bunch of them above, so would you mind changing these as well?